vars/isPluginInstalled.groovy

```diff
@@ -0,0 +1,26 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/**
+  Given the pluginName it validates whether it's installed and available.
+
+*/
+def call(Map args = [:]) {
+  def pluginName = args.pluginName
+  def plugins = jenkins.model.Jenkins.instance.getPluginManager().getPlugins().collectEntries { [(it.getShortName()):it.getVersion()] }
```

Comment:
I have a question about this syntax. Is there a reason that `{ [(it.getShortName()):it.getVersion()] }` can't be `{ [it.getShortName():it.getVersion()] }` (Removing the first pair of parenthesis?)

Reply:
Syntax problems:

```
Script1.groovy: 1: illegal colon after argument expression;
   solution: a complex label expression before a colon must be parenthesized @ line 1, column 100.
   tEntries { [it.getShortName() :it.getVer
                                 ^
```